command/resource/read/read_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+package read
+
+import (
+	"testing"
+
+	"github.com/mitchellh/cli"
+	"github.com/stretchr/testify/require"
+)
+
+func TestResourceReadInvalidArgs(t *testing.T) {
+	t.Parallel()
+
+	type tc struct {
+		args           []string
+		expectedCode   int
+		expectedErrMsg string
+	}
+
+	cases := map[string]tc{
+		"missing file path": {
```

Comment:
I think we are missing empty and nil args test cases in here.

Reply:
right! just as you pointed out the length sanity check! will add it here

Reply:
What about `nil` args?